web/src/components/Plugin/PluginPage.tsx

```diff
@@ -96,23 +97,32 @@ const PluginPage: React.FC<Props> = ({
       form.setFieldsValue({ plugin_config_id });
     });
   }, []);
+
+  useEffect(() => {
+    const openList = pluginList.filter(
+      (item) => initialData[item.name] && !initialData[item.name].disable,
+    );
+    setEnablePluginsList(openList);
+  }, [initialData]);
+
   const openPluginList = pluginList.filter(
     (item) => initialData[item.name] && !initialData[item.name].disable,
   );
+
   const openPluginType = openPluginList.map((item) => item.type);
   const newOpenPluginType = openPluginType.filter((elem, index, self) => {
     return index === self.indexOf(elem);
   });
 
-  const [showEnablePlugin, setShowEnablePlugin] = useState<boolean>(true);
+  const [showEnablePlugin, setShowEnablePlugin] = useState<boolean>(false);
```

Comment:
```suggestion
  const [showEnablePlugin, setShowEnablePlugin] = useState(false);
```
move this line to 71

Reply:
OK